src/Agents.jl

```diff
@@ -11,6 +11,8 @@ using Distributed
 using DataStructures
 using Graphs
 using DataFrames
+using ExproniconLite
+export @expr, JLKwStruct, JLKwField, codegen_ast
```

Comment:
```suggestion
```

I don't think we need to export these functions, right?

Reply:
For the way it's written now I think it's needed because the expr has to be evaluated in main scope and these functions need to be available. There might be a way around it though